src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: (currentResult) => (value) => currentResult + value,
```

Comment:
You need to work with `this` fix it everywhere
```suggestion
    add(value) {
      this.result += value;
    }
```